pandas/core/groupby/generic.py

```diff
@@ -1700,8 +1700,7 @@ def arr_func(bvalues: ArrayLike) -> ArrayLike:
                 "transform", bvalues, how, 1, **kwargs
             )
 
-        # We could use `mgr.apply` here and not have to set_axis, but
-        #  we would have to do shape gymnastics for ArrayManager compat
+        # TODO: With ArrayManager removed, we can now use mgr.apply
```

Comment:
Planning on leaving this for a followup, but can also take care of here

Reply:
Feel free to take care of it here

Reply:
Done.